app/lib/runtime/action-runner.ts

```diff
@@ -6,7 +6,8 @@ import { createScopedLogger } from '~/utils/logger';
 import { unreachable } from '~/utils/unreachable';
 import type { ActionCallbackData } from './message-parser';
 import type { BoltShell } from '~/utils/shell';
-import { createDefaultEnv } from '~/lib/webcontainer/defaultEnv';
+import { createDefaultEnv } from '~/lib/container/defaultEnv';
```

Comment:
`~/lib/container/defaultEnv` seems to replace `~/lib/webcontainer/defaultEnv`. if then, `~/lib/webcontainer/defaultEnv` should be deleted.